parser/parser_system1.c

```diff
@@ -230,6 +231,35 @@ static int parse_bsp_drivers(struct pv_state *s, char *v, int len)
 	return 1;
 }
 
+static pv_disk_format_t get_format(const char *str, jsmntok_t *diskv, int diskc)
```

Comment:
maybe rename these to parse_disks_get_format, parse_disks_get_type... to get less generic names